packages/react-components/components/KeyboardAwareScrollView.tsx

```diff
@@ -0,0 +1,96 @@
+import * as React from 'react'
```

Comment:
If we're implementing our own version of this, can we remove the old one from mobile's dependencies?

Reply:
We'll be able to remove the old one. I didn't want to touch too much yet 😄 
I'll open an issue.

Reply:
Opened #1247